CONTRIBUTING.md

```diff
@@ -2,19 +2,16 @@
 Our project welcomes external contributions. If you have an itch, please feel
 free to scratch it.
 
-To contribute code or documentation, please submit a pull request. 
-
-A good way to familiarize yourself with the codebase and contribution process is
-to look for and tackle low-hanging fruit in the issues. 
-Before embarking on a more ambitious contribution, please quickly [get in touch](#communication) with us.
+To contribute code or documentation, please submit a pull request. You can get started with open issues with the label - good first issue. 
```

Comment:
To contribute code or documentation, please submit a pull request. You can get involved by working on our open issues labeled 'good first issue'.

Reply:
Updated